src/library/scala/collection/Iterable.scala

```diff
@@ -758,7 +758,7 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
     *  @param that     the iterable providing the second half of each result pair
     *  @param thisElem the element to be used to fill up the result if this $coll is shorter than `that`.
     *  @param thatElem the element to be used to fill up the result if `that` is shorter than this $coll.
-    *  @return        a new collection of type `That` containing pairs consisting of
+    *  @return        a new collection of the type of this $coll containing pairs consisting of
```

Comment:
Looking around that file, most operations returning a `CC` say `@return a new $coll ...`, which seems easier to read.